apps/frontend/src/renderer/components/KanbanBoard.tsx

```
@@ -589,7 +654,14 @@ export function KanbanBoard({ tasks, onTaskClick, onNewTaskClick, onRefresh, isR
         onDragOver={handleDragOver}
         onDragEnd={handleDragEnd}
       >
-        <div className="flex flex-1 gap-4 overflow-x-auto p-6">
+        <div
+          ref={scrollContainerRef}
+          onMouseDown={handleMouseDown}
+          className={cn(
+            "flex flex-1 gap-4 overflow-x-auto p-6 cursor-grab select-none",
+            isScrollDragging && "cursor-grabbing active:cursor-grabbing"
+          )}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)